.env.development

```diff
@@ -2,3 +2,5 @@ FIND_BASE_URL=https://api.publish-teacher-training-courses.service.gov.uk/api/v3
 TEACHER_TRAINING_API_BASE_URL=https://api.publish-teacher-training-courses.service.gov.uk/api/public/v1
 HOSTING_ENVIRONMENT_NAME=development
 STATE_CHANGE_SLACK_URL=
+GA_APPLY=UA-APPLY-XX
```

Comment:
Could these be more descriptively named? Could we go with APPLY_GA_MEASUREMENT_ID and MANAGE_GA_MEASUREMENT_ID ?

Reply:
I think measurement_id makes it a bit long winded and confusing, as the analytics id is quite a commonly used property across projects, but maybe GOOGLE_ANALYTICS_APPLY and GOOGLE_ANALYTICS_MANAGE so it's more clear?

Reply:
Yeah, that would work too :)